### ISSUE TITLE
Need for Pull Request template

### ISSUE THREAD

**Issue by dostonnabotov** [dostonnabotov/quicksnip#41]
We need to make a simple pull request template for our contributors. It would make so much easier for our maintainers to review and debug problems as well.

I would love to hear some ideas on what to include and how to format the template, so it's easy to understand and don't overwhelm new/first-time contributors at the same time.

**Comment by Mathys-Gasnier** [comment]
We also need to inform them about not modifying the `all_snippets.json` file xD

**Comment by Mathys-Gasnier** [comment]
We should also have a convention for naming pull requests, i was thinking something along those line:
[Type] Name

So if for example we are modifying something in the css:
[CSS] Updating thing
And if we are adding, removing or updating a snippet:
[Snippet/Language] Name

**Comment by GreenMan36** [comment]
Something loosely along the lines of;

```markdown
# Description

<!-- Please include a summary of your changes. -->

## Type of Change

<!-- Please delete options that are not relevant and check the appropriate boxes. -->

- [ ] New snippet(s)
- [ ] Improvement to existing snippet(s)
- [ ] Bug fix
- [ ] Documentation update
- [ ] Other (please describe):

## Checklist

<!-- Please check all items that apply using "x" -->

- [ ] I have tested all submitted code snippets and verified they work as intended
- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code appropriately where needed
- [ ] I have updated the documentation accordingly (if applicable)
- [ ] My changes generate no new warnings
- [ ] For new snippets:
  - [ ] I have added appropriate tags
  - [ ] I have included a clear description
  - [ ] The category is appropriate (or I have created a new one)
  - [ ] Code is properly formatted and commented

## Related Issues

<!-- Please link to any related issues here using the GitHub syntax: #issue-number -->

Closes #

## Additional Context

<!-- Add any other context about the pull request here. -->

## Screenshots

<!-- If you made visual changes other than snippets, add screenshots to help explain your changes. -->

<details>
<summary>Screenshots</summary>

<!-- Paste your screenshots here -->

</details>
```
?

**Comment by Mathys-Gasnier** [comment]
That looks really good ! Thanks for the work on that front

**Comment by Mathys-Gasnier** [comment]
One other thing I would like, and i'm probably not alone on this... When adding a new language, even if the PR also adds some snippet, it should add the `Hello World!` snippet for that language

**Comment by dostonnabotov** [comment]
I just found that FreeCdoeCamp has done a really good job in creating issue templates:

![GitHub issue templates preview](https://github.com/user-attachments/assets/059f6353-0097-455d-bf77-d1d11a36a5fc)

A great place to take inspirations.

**Comment by dostonnabotov** [comment]
Now, we have a pull request template as well. Here's the first preview:

![Pull request template in use](https://github.com/user-attachments/assets/0786f51b-b092-48ee-ac61-cb942b7de0fc)

**Commit solved this issue**: 1d08bb2
